cypress/e2e/deputy_details.cy.js

```diff
@@ -123,4 +123,31 @@ describe("Deputy details tab", () => {
             ).should("contain", "some info for the pa deputy");
         });
     });
+
+    describe("Delete a deputy", () => {
+        it("can delete a deputy", () => {
+            cy.setCookie("success-route", "/deputies/1");
+            cy.get('[href="/supervision/deputies/1/delete-deputy"]').click();
+            cy.url().should("contain", "supervision/deputies/1/delete-deputy");
+            cy.get('.govuk-button').click();
+            cy.get('.moj-banner--success').should('be.visible');
+            cy.get('.moj-banner--success').should('contain.text', 'Test Organisation 11 has been deleted');
+        });
+
+        it("can't delete a deputy", () => {
+            cy.visit("/supervision/deputies/2/delete-deputy");
+            cy.setCookie("fail-route", "deleteDeputy");
+            cy.url().should("contain", "supervision/deputies/2/delete-deputy");
```

Comment:
This is unnecessary. You already know you are at this route as that is the one you told Cypress to visit.

Reply:
done